sw/device/tests/example_chip_level_test.c

```diff
@@ -0,0 +1,63 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/test_framework/ottf.h"
+
+const test_config_t kTestConfig = {
```

Comment:
Maybe say something to the effect that this is a specially-named symbol that the test framework looks for.

Reply:
Done.